.pre-commit-config.yaml

```diff
@@ -56,3 +56,11 @@ repos:
         language: golang
         require_serial: true
         files: ^kubectl-plugin/
+
+  - repo: local
+    hooks:
+      - id: validate-helm-charts
+        name: validate helm charts with kubeconform
+        entry: bash scripts/validate-helm.sh
+        language: system
+        pass_filenames: false
```

Comment:
Also check the version of `kubeconform` like here. Linters output usually differ for different versions so it is better to pin a specific version.

https://github.com/ray-project/kuberay/blob/22cc61d3dcfe70a4a0e84820c58175403f7d5c3f/.pre-commit-config.yaml#L30-L38